README.md

```diff
@@ -17,35 +18,14 @@ bash ./scripts/launch_validator_node.sh --node-name=TEST_NODE --network=testnet
 | Parameter name    | Required | Possible options             | Example                    | Description                                                                                                                                                |
 |-------------------|----------|------------------------------|----------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------|
 | node-name         | Yes      | Any string.                  | `--node-name=my-test-node` | A node name.                                                                                                                                               |
-| network           | Yes      | `testnet`, `testnet-dev`     | `--network=testnet`        | A network name.                                                                                                                                            |
+| network           | Yes      | `devnet`, `qanet`, `testnet`, `mainnet`     | `--network=testnet`        | A network name.                                                                                                                                            |
```

Comment:
In few rows, I see that `by default it will be taken from parameters`. Please, wrap word `parameters` with link to them.

Reply:
done